src/DacpacTool/PackageAnalyzer.cs

```diff
@@ -46,7 +46,7 @@ public void Analyze(TSqlModel model, FileInfo outputFile, FileInfo[] analyzers)
 
                 if (!rules.Any(r => r.Namespace == "SqlServer.Rules" || r.Namespace == "Smells"))
                 {
-                    _console.WriteLine("DacpacTool warning SQLPROJ0001: No additional well-known rules files found, consider adding more rules via PackageReference - see the readme here: https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/blob/master/README.md#static-code-analysis");
+                    _console.WriteLine("DacpacTool warning SQLPROJ0001: No additional well-known rules files found, consider adding more rules via PackageReference - see the readme here: https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/blob/master/README.md#static-code-analysis. You can ignore this warning with '<NoWarn>$(NoWarn);SQLPROJ0001</NoWarn>' in your project file.");
```

Comment:
Perhaps we should phrase this as "You can ignore this warning by adding ... to your project file"?